### PR TITLE
add the animation id of attaching bolt tips from bronze to rune

### LINKED ISSUE #11346: Idle notifier for adding bolt tips to darts
There is currently no idle notification for adding bolt tips to bolts when fletching 1,5 or 10 sets of bolts.

### PR DESCRIPTION
**Pull request by EtienneBoutet**
fix #11346

### CODE REVIEW

**Comment by EtienneBoutet** [comment]
I can't find anyone with 76+ fletching for the broad bolts and dragon bolts.. 
I'm pretty sure broad is 8479 and dragon is 8480.

Can't figure it out now.

**Comment by Nightfirecat** [comment]
Please test these ingame using the `::anim` dev tools command to confirm these are as expected--not that I would expect them to be otherwise, Jagex typically keeps similar animations adjacent to others in the ID ordering.

File: runelite-api/src/main/java/net/runelite/api/AnimationID.java [hunk 1]
```diff
@@ -74,6 +74,12 @@
 	public static final int FLETCHING_STRING_MAGIC_LONGBOW = 6689;
 	public static final int FLETCHING_ATTACH_FEATHERS_TO_ARROWSHAFT = 8481;
 	public static final int FLETCHING_ATTACH_HEADS = 8480;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_BRONZE_BOLT = 8472;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_IRON_BOLT = 8473;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_STEEL_BOLT = 8475;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_MITHRIL_BOLT = 8476;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_ADAMANT_BOLT = 8477;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_RUNE_BOLT = 8478;
```
Comment (Nightfirecat) [inline runelite-api/src/main/java/net/runelite/api/AnimationID.java:82]: Let's put these starting on line 75, after `FLETCHING_STRING_MAGIC_LONGBOW` and before `FLETCHING_ATTACH_FEATHERS_TO_ARROWSHAFT` to be closer to ID order.

**Comment by EtienneBoutet** [comment]
Thanks for your help, I've added the blurite and dragon bolts animation ids to both files.

File: runelite-api/src/main/java/net/runelite/api/AnimationID.java [hunk 2]
```diff
@@ -72,8 +72,16 @@
 	public static final int FLETCHING_STRING_YEW_LONGBOW = 6688;
 	public static final int FLETCHING_STRING_MAGIC_SHORTBOW = 6683;
 	public static final int FLETCHING_STRING_MAGIC_LONGBOW = 6689;
-	public static final int FLETCHING_ATTACH_FEATHERS_TO_ARROWSHAFT = 8481;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_BRONZE_BOLT = 8472;
+	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_IRON_BOLT = 8473;
```
Comment (Nightfirecat) [inline runelite-api/src/main/java/net/runelite/api/AnimationID.java:76]: This is the ID used for both iron and broad bolt tip attaching, so update this var's name.
```suggestion
	public static final int FLETCHING_ATTACH_BOLT_TIPS_TO_IRON_BROAD_BOLT = 8473;
```

**Pull request merged**